src/olympia/versions/templates/versions/version_list.html

```diff
@@ -26,7 +26,7 @@
       <h1>{{ _('{0} Version History')|fe(addon.name) }}</h1>
       {% with cnt=versions.paginator.count %}
         {# L10n: {0} is a number. #}
-        <h3>{{ ngettext('<b>{0}</b> version', '<b>{0}</b> versions', cnt)|f(cnt)|safe }}</h3>
+        <h3>{{ ngettext('<b>{0}</b> version', '<b>{0}</b> versions', cnt)|fe(cnt)|safe }}</h3>
```

Comment:
unneccessary, not redundant :laughing: unfortunately there's a different semantic here but yeah, given that `cnt ` should be safe and if not should be escaped `|safe` is probably not cool here.